decidim-surveys/lib/decidim/surveys/component.rb

```diff
@@ -163,7 +163,7 @@
 
       3.times do
         question.answer_options.create!(body: Decidim::Faker::Localized.sentence)
-        question.matrix_rows.create!(body: Decidim::Faker::Localized.sentence)
+        question.matrix_rows.create!(body: Decidim::Faker::Localized.sentence, position: index)
```

Comment:
💬 Just to be sure, this is creating 3 rows with the same position, is this ok?

Reply:
You're right @leio10 matrix_rows need their specific position index. Changed!